makelib/golang.mk

```diff
@@ -281,11 +315,10 @@ vendor: go.vendor
 vendor.check: go.vendor.check
 vendor.update: go.vendor.update
 vet: go.vet
-generate codegen: go.generate
 
 define GO_HELPTEXT
 Go Targets:
-    generate        Runs go code generation followed by goimports on generated files.
+    go.generate     Runs go code generation followed by goimports on generated files.
```

Comment:
I think we should leave the original target name to not break projects/flow.

Reply:
then how should we resolve the target collision with the new `generate` target that we are adding in kubebuilder.mk?

Reply:
this will make the generate targets  in `golang.mk` and `kubebuilder.mk` to collide which results in executing both and produces an error. I'll rename the target in kubebuilder to avoid the collision.